CHANGELOG.md

```diff
@@ -1,6 +1,102 @@
 CHANGELOG
 =========
 
+v2.0.0
+-------------
+
+### Summary
```

Comment:
Please also match `vector-datasource` changelog conventions here:

```
**Release date:** 2017-12-06.
**Requires:** tileserver v3.0.0 and vector-datasource v1.8.1 and mapbox-vector-tile v1.2.0.
```

(Updating the requires versions as appropriate...)